manifests/charts/istio-operator/templates/deployment.yaml

```diff
@@ -23,6 +23,9 @@ spec:
 {{ toYaml .Values.podAnnotations | indent 8 }}
     {{- end }}
     spec:
+      {{- if .Values.priorityClassName }}
```

Comment:
I think this can be done with https://istio.io/latest/docs/setup/additional-setup/customize-installation-helm/?

Reply:
yes thats how we currently do it but it feels dirty/custom to do it this way instead of just having a proper field available in values

Reply:
for anyone wondering we are doing it this way with helmfile.yaml:
```
repositories:
  - name: istio
    # see this issue for why we cannot use annotated tags with sparse=1
    # https://github.com/aslafy-z/helm-git/issues/9
    # https://github.com/aslafy-z/helm-git/pull/226
    # commit for 1.16.7
    url: git+https://github.com/istio/istio@manifests/charts?ref=00136097767eec680da8749837a8d79adf110b1a&sparse=1

releases:
  - name: istio-operator
    <<: *base
    chart: istio/istio-operator
    # version never changes, its bound to the app version
    # so change the app version in the repository ref instead
    version: 1.0.0
    jsonPatches:
      - patches/deployment.yaml
```
patches/deployment.yaml:
```
patch:
  - op: add
    path: "/spec/template/spec/priorityClassName"
    value: "platform-critical"
target:
  version: v1
  kind: Deployment
  name: istio-operator
```